### PR TITLE
Rholang: Fix typo introduced in fe1d720a

### PR DESCRIPTION
**Pull request by guardbotmk3**

### CODE REVIEW

File: rholang/src/main/scala/rholang/rosette/Compiler.scala [hunk 1]
```diff
@@ -55,7 +55,7 @@ object Rholang2RosetteCompiler extends RholangASTToTerm
         None
       }
       case t : Throwable => {
-        System.err.println(s"""Error while compliling: ${fileName}\n${e.toString()}""")
+        System.err.println(s"""Error while compliling: ${fileName}\n${t.toString()}""")
```
Comment (kirkwood) [inline rholang/src/main/scala/rholang/rosette/Compiler.scala]: Could do "compliling" while you're at it :)

**Comment by dckc** [comment]
Did fe1d720 get merged? If so, does that mean travis is not finding build failures?

**Comment by KentShikama** [comment]
@dckc This is only a warning not an error so it continues building.

**Comment by echatav** [comment]
@KentShikama No, it's a build error, at least for me
```
[error] /Users/hephaestus/Documents/GitHub/rchain/rholang/src/main/scala/rholang/rosette/Compiler.scala:58:71: not found: value e
[error]         System.err.println(s"""Error while compliling: ${fileName}\n${e.toString()}""")
[error]                                                                       ^
[error] one error found
[error] (compile:compileIncremental) Compilation failed
```

**Comment by echatav** [comment]
Wrote another PR, #170, to fix this issue as well as a couple others, so can we merge that one instead?

**Comment by guardbotmk3** [comment]
Sorry for the delay.

**Pull request merged**